plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/resource/ResourcesController.java

```diff
@@ -111,10 +116,10 @@ public ResourcesController(ResourcesService service, ResourcesValidator resource
             @ApiResponse(code = 401, message = "Unauthorized")})
     @DeleteMapping("/plugins/cms/assets/{resourceId}")
     @RestAccessControl(permission = Permission.CONTENT_EDITOR)
-    public ResponseEntity deleteAsset(@PathVariable("resourceId") String resourceId) {
+    public ResponseEntity<SimpleRestResponse<Void>> deleteAsset(@PathVariable("resourceId") String resourceId) {
```

Comment:
This is kind of interesting. What kind of response does this generate? Does this create the outer structure

Reply:
```
{
    "payload": null,
    "metaData": {},
    "errors": []
}
```

Reply:
Instead of `Void` we could pass `Object` so the response would look like:
```
{
    "payload": {},
    "metaData": {},
    "errors": []
}
```